pantheon-bundle/src/test/java/com/redhat/pantheon/servlet/module/ModuleVersionUploadTest.java

```diff
@@ -49,7 +74,7 @@ void createFirstVersion() throws Exception {
         // Then
         assertEquals(HttpServletResponse.SC_CREATED, response.getStatusCode());
         assertNotNull(slingContext.resourceResolver().getResource("/content/repositories/test_workspace/entities/new/proc_module/es_ES/source/draft/jcr:content"));
-        assertNull(slingContext.resourceResolver().getResource("/content/repositories/test_workspace/entities/new/proc_module/es_ES/variants"));
+        assertNotNull(slingContext.resourceResolver().getResource("/content/repositories/test_workspace/entities/new/proc_module/es_ES/variants"));
```

Comment:
a comment for all these tests:
we should at least add `verify` calls to ensure the appropriate methods are being called in the `asciidoctorService` and `serviceResourceResolvedProvider` objects